docs/applications/nublado/updating-recommended.rst

```diff
@@ -0,0 +1,44 @@
+##############################################
+Updating the recommended Notebook Aspect image
+##############################################
+
+The ``recommended`` tag for JupyterLab images is usually a recent weekly image.
+The image tagged ``recommended`` is guaranteed by SQuaRE to be compatible with other services and materials, such as tutorial or system testing notebooks, that we make available on RSP deployments.
+
+Because this process requires quite a bit of checking and sign-off from multiple stakeholders, it is possible that approving a new recommended version may take more than the two weeks (for most deployments) it takes for a weekly image to roll off the default list of images to pull.
+This can cause the RSP JupyterHub options form to display empty parentheses rather than the correct target version when a user requests a lab container.
+
+This document explains the process for moving the ``recommended`` tag, and how to circumvent that display bug by changing nublado's ``values-<instance>.yaml`` for the appropriate instance when moving the ``recommended`` tag.
+
+Tagging a new container version
+--------------------------------
+
+When a new version is to be approved (after passing through its prior QA and sign-off gates), the ``recommended`` tag must be updated to point to the new version.
+
+To do this, run the GitHub retag workflow for the `sciplat-lab <https://github.com/lsst-sqre/sciplat-lab>`__ repository, as follows:
+
+#. Go to `the retag workflow page <https://github.com/lsst-sqre/sciplat-lab/actions/workflows/retag.yaml>`__.
+#. Click on :guilabel:`Run workflow`.
+#. Enter the tag of the image to promote to recommended under :guilabel:`Docker tag of input container`.
+   This will be a tag like ``w_2022_40``.
+#. Enter ``recommended`` under :guilabel:`Additional value to tag container with`.
+#. Click on the :guilabel:`Run workflow` submit button.
+
+Don't change the URIs.
```

Comment:
Maybe move this up into the list, as you're talking about all the configuration options.  The last thing should be "Run workflow"

Reply:
also maybe say don't change "fully-qualified URI for output Docker image"  since you're giving all the full text for each of the form options